File: Iniciante/1044 - Múltiplos/Python/1044.py
def main():
    a, b = map(int, input().split())
    result = are_multiples(a, b)
    print(result)


def are_multiples(a, b):
    if (a != 0 and b % a == 0) or (b != 0 and a % b == 0):
        return "Sao Multiplos"
    else:
        return "Nao sao Multiplos"


if __name__ == "__main__":
    main()

File: Iniciante/1044 - Múltiplos/JavaScript/1044.js
function main() {
  const input = require("fs").readFileSync("/dev/stdin", "utf8");
  const [a, b] = input.split(" ").map(Number);
  const result = areMultiples(a, b);
  console.log(result);
}

function areMultiples(a, b) {
  if ((b % a === 0) || (a % b === 0)) {
    return "Sao Multiplos";
  } else {
    return "Nao sao Multiplos";
  }
}

main();
